### DISCUSSION TITLE
Deployment Error on GitHub Pages with gh-pages branch

### DISCUSSION THREAD

**Discussion by jbloch100** [community/community#118745] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I’m encountering a deployment error when trying to publish my site to the gh-pages branch. The build completes successfully, but deployment fails with an error ‘timeout reached’.

**Comment by LiteBrite82** [comment]
Can you provide a link to your repo?

**Reply by jbloch100** [reply, marked as answer]
https://github.com/jbloch100/travel-companion

Like I got it to work but I don't know how though

**Reply by LiteBrite82** [reply]
Im glad it worked! Nice app by the way! Clean coding, easy to understand, great documentation! 

Since Create React App isn't being supported anymore, have you looked into other React frameworks? I transitioned to Vite last year; there is a bit of a learning curve.

It has some significant advantages though. The build times are incredibly fast compared to CRA, and it runs in real time; no need to reload with code changes. 

Thanks for sharing @jbloch100 !

**Reply by jbloch100** [reply]
I'll definitely look into it. Thanks for the update
